src/python/zigopt/common/sigopt_datetime.py

```diff
@@ -55,6 +55,10 @@ def unix_timestamp(with_microseconds: bool = False) -> int | float:
   return datetime_to_seconds(current_datetime(), with_microseconds=with_microseconds)
 
 
+def unix_timestamp_seconds() -> int:
```

Comment:
do we need this over `unix_timestamp()` which would also return an `int` (in that case)? Is it a stronger promise (no chance of `float`) that's important / helpful for typing somewhere?

Reply:
Yes, a function that only returns a single type is better for typing since mypy cant infer the return type of the function from the arguments. I thought splitting out `with_microseconds` would be too large a diff, but it looks like we rarely use it so I'll just include that change here.

Reply:
Yeah I split out these functions when I added typing to `zigopt.common` for this exact reason.